modules/transport-netty4/src/main/java/org/elasticsearch/http/netty4/Netty4HttpPipeliningHandler.java

```diff
@@ -477,6 +478,16 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
         }
     }
 
+    @Override
+    public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exception {
+        if (evt instanceof SslCloseCompletionEvent closeEvent) {
+            if (closeEvent.isSuccess() && ctx.channel().isActive()) {
+                logger.warn("received TLS close_notify, closing connection {}", ctx.channel());
```

Comment:
Is this actually a log worthy event?

Aren't there some clients that always have this mode of behavior which means we will we introducing many warn logs that are not indicating a problem? I'll defer to @DaveCTurner here, but I would assume we would not want this to be always logged.

Reply:
I think not worthy to be warn. I'm thinking Trace level might be more appropriate.